hud-data/goog-hud.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "googHUD",
+  "author": "grey",
+  "contributors": [
+    "grey"
+  ],
+  "releaseDate": "9/21/2023",
+  "flags": [
+    "menus",
+    "scoreboards",
+    "materials"
+  ],
+  "tags": [
+    "minimal",
+    "clean",
+    "competitive",
+    "comp",
+    "wagoogus",
+    "beta",
+    ""
+  ],
+  "repo": "https://github.com/GreySux/googhud-main",
+  "hash": "00d40765d044b014651d93cfd28db2780b9069fc",
+  "prerelease": true,
+  "resources": [
+    "https://www.youtube.com/watch?v=C4lgL-YI43U",
+    "cibahud",
```

Comment:
These should be names of screenshot files, it's fine to only have 1 image file which is a logo or a single screenshot, and have the rest in your imgur album.

Reply:
Hi, sorry I'm new to this and I'm a bit confused on what you mean by this. Should I be uploading the image file somewhere and putting the link in? or uploading the image file directly on github somewhere? or just putting the name of the pic?

Reply:
Example: https://github.com/mastercomfig/hud-db/tree/main/hud-resources/7hud

You can upload any image and it will convert to webp.

Reply:
More info: https://github.com/mastercomfig/hud-db#hud-resources